### ISSUE TITLE
Realms dying because of the difference between any realm and ray

### ISSUE THREAD

**Issue by Cykne** [SmashMC-Development/Suggestions#982]
I suggest limiting the amount of people that can go in each realm (especially ray) so all the servers are actually playable because idk about the other realms but zekrom is dying, only 4-7 people usually are playing on zekrom while ray has about 50-70, sometimes more. This is basically making it impossible to sell anything on gts, difficult to have fun, and overall making me and probably other players on zekrom, less motivated to actually play pixelmon.

Limiting the amount of people should be done in each realm.

N/A

**Comment by morrow12** [comment]
imo not a good idea it's up to the players to play what server they want

**Comment by Cykne** [comment]
> imo not a good idea it's up to the players to play what server they want

Yes i agree but what even is the point of the other realms if they're barely even being played, I might just quit from zekrom and go to ray just because of how more lively it is, and probably more people will do in the future resulting in literally 0 people playing on other realms and ray being overpopulated asf

**Comment by XITACE** [comment]
True
